README.md

```diff
@@ -27,13 +27,12 @@ Make sure to save your changes before exiting the settings page.
 > [!NOTE]
 > 
 > Granting `Read and write permissions` allows workflows to modify your repository, which can include adding or updating files and code. Always ensure that you trust the workflows you enable with these permissions.
- 
-
-![General Settings](docs/images/Github_Initial_Git_Push_Settings_General.png)
```

Comment:
We can take out the picture, but I would suggest that we describe the location of the settings better.

Reply:
I decided to change the picture.